tools/eksDistroBuildToolingOpsTools/Makefile

```diff
@@ -1,14 +1,35 @@
 GIT_VERSION?=v0.0.0-dev
+IMAGE_BUILD_ID=$(shell cat ./EKS_DISTRO_PROW_PLUGIN_RELEASE)
+
+AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
+
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+PROJECT_DIRECTORY:=$(BASE_DIRECTORY)/tools/eksDistroBuildToolingOpsTools
+
+# This image build is specifically focused on a debian base image for use with EKS build systems
+IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+PUSH_IMAGES?=true
+
+GITHUB_EMAIL?="prow@amazonaws.com"
+GITHUB_USER?="Prow Bot"
 
 GO_ARCH:=$(shell go env GOARCH)
 GO_OS:=$(shell go env GOOS)
 
 EKS_GO_TOOLS_BINARY_NAME="eksGoTools"
+EKS_PROW_PLUGIN_BINARY_NAME="eksDistroOpsProwPlugin"
 
 .PHONY: build-eks-go-tools
-build-eks-go-tools:  unit-test lint
+build-eks-go-tools:
 	CGO_ENABLED=0 GOOS=$(GO_OS) GOARCH=$(GO_ARCH) go build -ldflags "-X github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/version.gitVersion=$(GIT_VERSION) -s -w -extldflags -static" -o bin/$(GO_OS)/$(GO_ARCH)/$(EKS_GO_TOOLS_BINARY_NAME) github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool
 
+.PHONY: build-prowPlugin
+build-prowPlugin: 
```

Comment:
let's run `lint` and `unit-test` as pre-reqs for this target, and remove them from the images target